manifest/io.go

```diff
@@ -16,7 +22,23 @@ func (m *Manifest) ToJson() ([]byte, error) {
 
 func ManifestFromYaml(content []byte) (manifest Manifest, err error) {
 	err = yaml.Unmarshal(content, &manifest)
-	return
+	if err != nil {
+		return manifest, err
+	}
+
+	return manifest, manifest.Validate()
+}
+
+func (m Manifest) Validate() error {
+	for _, mount := range m.Mounts {
+		if mount.LocalDir != "" {
+			if !filepath.IsAbs(mount.LocalDir) {
+				return fmt.Errorf("localDir needs to be absolute path")
```

Comment:
I'm trying to think whether this constraint is absolutely necessary. I guess it's better to be strict from the start.

Reply:
Yea it requires less logic if the path is absolute to begin with.